mmv1/third_party/terraform/services/compute/resource_compute_disk_test.go.erb

```diff
@@ -560,7 +560,6 @@ func TestAccComputeDisk_encryptionKMS(t *testing.T) {
 	})
 }
 
-<% unless version == 'ga' -%>
 func TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute(t *testing.T) {
 	t.Skip()
```

Comment:
Done. I updated the test as well to use regional key. Since the key service for confidential disk is only regional right now. I think this was the reason the test was skipped initially.